src/main/java/net/consensys/orion/cmd/Orion.java

```diff
@@ -117,6 +118,7 @@ public static void configureRoutes(
       ConcurrentNetworkNodes networkNodes,
       Enclave enclave,
       Storage<EncryptedPayload> storage,
+      Storage<String[]> privacyGroupStorage,
```

Comment:
Is the `EncryptedPayload` encrypted at rest? If so, then leaking the privacy group mapping is not a good idea.

Reply:
It is not encrypted at rest so ignore this